src/misc/index.ts

```diff
@@ -40,13 +40,13 @@ export namespace Color3 {
 /**
  * @alpha
  */
-export type WearableId = string
+export type WearableId = string // `urn:decentraland:${string}`
```

Comment:
does it make sense to add schema validation over this pattern?

Reply:
Now that you mention it, not really. We may add `urn:crytptovoxels:` or other interoperable thing in the future.
```suggestion
export type WearableId = string // `urn:${string}`
```